From a5e50893a2c208d4c56cd3d4df39acfac47a8a28 Mon Sep 17 00:00:00 2001
Message-ID: <a5e50893a2c208d4c56cd3d4df39acfac47a8a28.1750031643.git.calvin@wbinvd.org>
From: Calvin Owens <calvin@wbinvd.org>
Date: Sat, 14 Jun 2025 23:52:43 -0700
Subject: [PATCH 1/8] Tweak x86_64 makefile to support compilers other than gcc

Upstream-Status: Submitted [https://github.com/memtest86plus/memtest86plus/pull/521]
Signed-off-by: Calvin Owens <calvin@wbinvd.org>
---
 build/x86_64/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/build/x86_64/Makefile b/build/x86_64/Makefile
index 1eed8c2..76fe2b0 100644
--- a/build/x86_64/Makefile
+++ b/build/x86_64/Makefile
@@ -1,8 +1,8 @@
-AS = as -64
-CC = gcc
-OBJCOPY = objcopy
+AS ?= as -64
+CC ?= cc
+OBJCOPY ?= objcopy
 
-GIT = git
+GIT ?= git
 
 ifeq ($(GIT),none)
   GIT_AVAILABLE = false
-- 
2.47.2

